### PR TITLE
[Serializer] By default the serializer do not convert to lower case properties

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍 

@lyrixx thanks for fixing this error! I added a small example to show that the conversion doesn't change the lower/uppercase of the method name.

**Comment by lyrixx** [comment]
@javiereguiluz Thanks but actually it a bit more complex than that. The ObjectNormalizer use the property name for the name, but use the getter to get the value.


```php
class Post
{
    private $publishedAt;
    
    public function getPublishedAt()
    {
        return $this->publishedAt;
    }
}
```

it will be serialized to `{"publishedAt": "2017...."}"`

**Comment by javiereguiluz** [comment]
@lyrixx OK. Then I don't understand this phrase:

```
Objects are normalized to a map of property names (method name stripped of the get/set/has/remove prefix) to property values.
```

By the way,  I can understand your comment perfectly ... so maybe that's the description we should use.

**Comment by lyrixx** [comment]
Indeed, I don't understand the comment either...
maybe @dunglas could help us on this one.

**Comment by lyrixx** [comment]
May be we could explain how to add this name converter in the SE:

```
    serializer:
        enabled: true
        name_converter: serializer.name_converter.camel_case_to_snake_case
```

**Comment by HeahDude** [comment]
ping @dunglas :)

**Comment by lyrixx** [comment]
ping @dunglas

**Comment by javiereguiluz** [comment]
Should we close here? We have no more feedback or information about what to do 😞

**Comment by lyrixx** [comment]
No, you can merge as it

File: components/serializer.rst [hunk 1]
```diff
@@ -512,7 +512,7 @@ There are several types of normalizers available:
     calling the constructor during the denormalization process.
 
     Objects are normalized to a map of property names (method name stripped of
-    the "get"/"set"/"has"/"remove" prefix and converted to lower case) to property
```
Comment (dunglas) [inline components/serializer.rst]: the **first letter* is lower cased (getFoo -> foo)

Comment (dunglas) [reply components/serializer.rst]: So may I suggest: "Objects are normalized to a map of property names (method name stripped of the "get"/"set"/"has"/"remove" prefix and first letter converted to lower case) ..."

**Review by dunglas — APPROVED** [review]
👍, it's "obvious" that the conversion will occur

**Comment by javiereguiluz** [comment]
Thanks Grégoire and Kévin! Merged on 2.7 branch.

**Pull request closed without merging**